test/node/bigint.test.ts

```diff
@@ -263,4 +263,92 @@ describe('BSON BigInt support', function () {
       expect(serializedMap).to.deep.equal(expectedSerialization);
     });
   });
+
+  describe('EJSON.stringify()', function () {
```

Comment:
I'm going to leave two general test organization comments here, both of which boil down to "context blocks are great" 😄 

---

It can be really useful to group by behavior using context blocks.  There are many ways to do it (you can group by option, by functionality, ect).  I think in this case, grouping by option makes sense.  I would group any tests specific to canonical ejson and any tests specific to relaxed ejson together.  For example (not a complete refactor, just an example):

```typescript
context('when relaxed=true (canonical mode)', function() {
  it('truncates bigint values when they are outside the range [BSON_INT64_MIN, BSON_INT64_MAX]', function () { ... });
  it('truncates bigint values in the same way as BSON.serialize', function () { ... });
});
```

This

- does not clutter each test description with all the conditions that make up the test
- related to the above, this makes the output of the tests much easier to read
- can help identify code paths you might miss otherwise.  when you group by options / conditions, it's easy to see if you're missing a condition

That's the first change I'd suggest.

---

Related to the above, the "condition" should always come before the "expectation" in test output and in test structure (that's my opinion but one I hold rather strongly).  I think contexts make this easy because they break the condition apart from the expectation.  The example, the following is easier to grok than a single test with the description "produces bigint strings that are equal to the strings generated when using BigInt.toString when the bigint values used are 64 bits wide or less":

```typescript
context('when the bigint values used are 64 bits wide or less', function() {
  it('produces identical values to `BigInt.toString`', () => { ... });
});
```

Grouping by condition (my first point) helps here, but I always use contexts even when I have a single test because it helps with clarity.